weblate_web/templates/snippets/users.html

```diff
@@ -1,10 +1,13 @@
 {% load i18n %}
 {% load static %}
+<a href=""><img src="{% static "img/Logo-Turris.png" %}" alt="Turris" /></a>
+<a href=""><img src="{% static "img/LogoSonicPi.png" %}" alt="SonicPi" /></a>
 <a href="https://l10n.opensuse.org/"><img src="{% static "img/logo-opensuse.svg" %}" alt="openSUSE" /></a>
+<a href=""><img src="{% static "img/LogoPhpMyAdmin.png" %}" alt="PhpMyAdmin" /></a>
+<a href=""><img src="{% static "img/LogoFreedomBox.png" %}" alt="FreedomBox" /></a>
 <a href="https://translations.documentfoundation.org/"><img src="{% static "img/logo-libreoffice.svg" %}" alt="LibreOffice" /></a>
 <a href="https://translate.fedoraproject.org/"><img src="{% static "img/logo-fedora.svg" %}" alt="Fedora" /></a>
-<a href="https://hosted.weblate.org/engage/freedombox/"><img src="{% static "img/logo-freedombox.svg" %}" alt="Freedombox" /></a>
 <a href="https://hosted.weblate.org/projects/collabora-online/"><img src="{% static "img/logo-collabora.svg" %}" alt="Collabora" /></a>
 <div class="clear"></div>
-<a href="https://hosted.weblate.org/projects/" class="button border small rev inline" target="_blank">{% trans "See all projects" %}</a>
-<a href="{% url 'donate' %}#thanks" class="button border small rev inline">{% trans "See all supporters" %}</a>
+<a href="https://hosted.weblate.org/projects/" class="button border small rev inline" style="text-decoration: none;" target="_blank">{% trans "See all projects" %}</a>
```

Comment:
Please avoid inline CSS (this one should not be necessary, the styling should be done by the classes).

Reply:
Yes, the inline CSS has been removed.

Reply:
I can still see it in this place (`style="text-decoration: none;"`).

Reply:
I apologize for this, i have gotten rid of that now.